src/constants/urls.ts

```diff
@@ -0,0 +1,2 @@
+export const DISCORD_INVITE_LINK = 'https://discord.com/invite/QFkNsjTkzD'
+export const LIQUIDITY_V3_LINK = 'https://swapr.liquidity.eth.limo/'
```

Comment:
Can we define this as env variables with fallback?

Reply:
Good idea, will do it now!